code/game/turfs/simulated.dm

```diff
@@ -165,3 +165,12 @@
 	if(GAME_STATE >= RUNLEVEL_GAME)
 		fluid_update()
 	. = ..()
+
+/turf/simulated/Destroy()
```

Comment:
This isn't a GC return value fix. Document if intentional, pls.

Reply:
ok, it's in the PR message now.